include/fastdds/dds/domain/DomainParticipantListener.hpp

```diff
@@ -132,9 +132,11 @@ class DomainParticipantListener :
      * This method is called when a new Subscriber is discovered, or a previously discovered subscriber changes
      * its QOS or is removed.
      *
+     * @warning Not Supported. This callback will never be called in the current version.
+     *
      * @param[out] participant Pointer to the Participant which discovered the remote subscriber.
      * @param[out] info Remote subscriber information. User can take ownership of the object.
-     * @param[out] should_be_ignored Flag to indicate the library to automatically ignore the discovered Participant.
+     * @param[out] should_be_ignored Flag to indicate the library to automatically ignore the discovered subscriber.
```

Comment:
Maybe we should update the entity name to avoid misunderstandings. Instead of subscriber/publisher, use DataReader/DataWriter (even if the callback name is maintained for backwards compatibility sake).

Reply:
I used subscriber/publisher because the rest of the comment, and other comments only mention those, and never DataReader/DataWriter.
I agree that DR/DW would make more sense, but I would change it in more places.
If you give me green light I make the change.

Reply:
:green_circle: